source/documentation/guides/api_standards.html.md.erb

```diff
@@ -38,6 +38,11 @@ Adopt these standards if there is no reason not to, but you may break any of the
 
 For further information, see the [OPG Data versioning strategy](https://github.com/ministryofjustice/opg-data/blob/main/docs/architecture/supporting-notes/versioning-strategy.md)
 
+## Documentation
+
+1. Use the [OpenAPI Specification](https://swagger.io/specification/) for documenting APIs
+1. Store in the repo at `/docs/openapi/openapi.yml`
```

Comment:
We have some at this location, some at `/swagger.yml`, and some [others](https://github.com/ministryofjustice/opg-data-lpa/blob/main/lambda_functions/v1/openapi/lpa-openapi.yml). Under `/docs` I think makes the most sense.